tests/support/graphQlGetCollections.js

```diff
@@ -0,0 +1,5 @@
+/**
+ * Return the GraphQl query for verifying the request is what we expect inside of cy.intercept
```

Comment:
Gotcha just fix the doc-string

```suggestion
 * Return the GraphQl query for verifying the request is what we expect inside of page.route
```